tests/unit/test_terminal_utils.py

```diff
@@ -1,4 +1,8 @@
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
```

Comment:
Can you remove this? It would be better to have the pull request do one thing at a time.

Also, we already have a pull request to switch to `unittest.mock` directly, rather than conditionally: https://github.com/osrf/osrf_pycommon/pull/74

Reply:
Done.